services/121-service/test/performance/GetProgramWithManyAttributes.test.ts

```diff
@@ -0,0 +1,86 @@
+import { HttpStatus } from '@nestjs/common/enums/http-status.enum';
+import { performance } from 'node:perf_hooks';
+import { env } from 'process';
+
+import { ProgramRegistrationAttributeDto } from '@121-service/src/programs/dto/program-registration-attribute.dto';
+import { RegistrationAttributeTypes } from '@121-service/src/registration/enum/registration-attribute.enum';
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { registrationVisa } from '@121-service/src/seed-data/mock/visa-card.data';
+import {
+  getProgram,
+  postProgramRegistrationAttribute,
+} from '@121-service/test/helpers/program.helper';
+import {
+  duplicateRegistrations,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { programIdOCW } from '@121-service/test/registrations/pagination/pagination-data';
+
+const duplicateNumber = parseInt(env.DUPLICATE_NUMBER || '5'); // cronjob duplicate number should be 2^5 = 32
+
+describe('Get program with many attributes within time threshold', () => {
+  let accessToken: string;
+
+  it('Should get program with many attributes within time threshold', async () => {
```

Comment:
I don't see an explicit time threshold set here. Does that imply a default? (of 60s?) Maybe make that explicit given it's apparently important in this test.